website_logo/views/res_config_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record id="res_config_settings_view_form" model="ir.ui.view">
+    <field name="name">Adding website logo</field>
+    <field name="model">res.config.settings</field>
+    <field name="inherit_id" ref="website.res_config_settings_view_form"/>
+    <field name="priority" eval="21"/>
+    <field name="arch" type="xml">
+        <xpath expr="//div[@id='domain_setting']//div[hasclass('text-muted')][1]" position="attributes">
+            <attribute name="invisible">1</attribute>
+        </xpath>
+        <xpath expr="//div[@id='domain_setting']//div[hasclass('text-muted')][1]" position="after">
+            <div class="text-muted">
+                Name, favicon and logo of your website
+            </div>
+        </xpath>
+        <xpath expr="//div[@id='domain_setting']//field[@name='favicon']/.." position="after">
```

Comment:
shouldn't we state that `png` format is required?

Reply:
That's not requiredf AFAIK